### PR TITLE
[chore] Add flake8 workflow 

### LINKED ISSUE #226: Add tests to follow PEP8 guidelines
### Is your feature request related to a problem? Please describe.
PEP stands for Python Enhancement Proposal. It is a style document that documents some best practices

### Describe the solution you'd like
Code written in Python should follow standard practices and PEP8 guidelines. For this a linter that follows these guidelines or some sort of checks can be implemented

### Describe alternatives you've considered
I would suggest using Flake8 to test if PEP8 best practices are followed in the code written or not. Additionally, Flake8 workflow tests can be setup to ensure pull requests made follow these guidelines as well.

### Additional context
Flake8 is not the only tool available and other alternatives can be considered.

### PR DESCRIPTION
**Pull request by Aryamanz29**
### Description

Added workflow for Flake8

Fixes #226

### Type of Change:

- Code
- Quality Assurance

### How Has This Been Tested?
Tested on Github.

Fail : https://github.com/Aryamanz29/bridge-in-tech-backend/actions/runs/728480082
Success : https://github.com/Aryamanz29/bridge-in-tech-backend/actions/runs/728482243

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: flake8-check [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    flake8 && echo "Flake8 Check ✅" ||{
+        >&2 echo "Flake8 Check ❌"
+        echo "Suggestion: Run - flake8 ."
+        echo ""
+        FAILURE=1
+    }
+}
+
+flake8-check
+exit $FAILURE
```
Comment (codesankalp) [inline flake8-check:13]: a new line at the end is required.

File: .flake8 [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = F401,F405,W503,F841,F403,E402,F821,W291,E266,F811
```
Comment (codesankalp) [inline .flake8:2]: Why so much error code in ignoring. If there are errors then they should be fixed in this PR.
You can consider trailing commas and import in `__init__.py` error code in ignoring or any other code which is required but do mention about code.

**Review by codesankalp — CHANGES_REQUESTED** [review]
@Aryamanz29 It will be better if you add info about this in README.

**Comment by Aryamanz29** [comment]
> @Aryamanz29 It will be better if you add info about this in README.

@codesankalp  should I mention error codes `.flake8` itself or in `README`?

File: .flake8 (hunk 2 above)
Comment (Aryamanz29) [reply .flake8:2]: @codesankalp  should I mention error codes `.flake8` itself or in `README`?

Comment (codesankalp) [reply .flake8:2]: No just add how to run check for flake8.

**Comment by Aryamanz29** [comment]
Please don't review this PR, I'm still working on this. I'll ping maintainers soon after fixing tests.

**Comment by devkapilbansal** [comment]
@Aryamanz29 you can mark a pull request as draft if it is not ready for review. Btw, thanks for letting us know that this PR is still in progress

**Comment by Aryamanz29** [comment]
@devkapilbansal  While testing my code locally most `unittest` fails and I'm unable to debug it after several attempts, So kindly unassign this issue to me for now. I'll try to work on this issue later.


Screenshot (49)

**Comment by mtreacy002** [comment]
@Aryamanz29 , have you completed the local setup #244 as requested [here](https://github.com/anitab-org/bridge-in-tech-backend/issues/226#issuecomment-817141590)? If so, can you submit a pr showing the gif in the pr description as requested on the tasks?

**Comment by mtreacy002** [comment]
Unassigning as per contributor's request [here]https://github.com/anitab-org/bridge-in-tech-backend/pull/260#issuecomment-816832563).

**Pull request closed without merging**